bash_completion

```diff
@@ -1737,7 +1737,7 @@ _comp_compgen_available_interfaces()
         fi
     } 2>/dev/null | _comp_awk \
         '/^[^ \t]/ { if ($1 ~ /^[0-9]+:/) { print $2 } else { print $1 } }')" &&
-        _comp_compgen -U generated set "${generated[@]}"
+        _comp_compgen -U generated set "${generated[@]%%[[:punct:]]*}"
```

Comment:
On a second thought, this is not correct, it will break display of things like `br-something` and `lo:0` (aliases only come out from `ifconfig` that way it seems).